docs/data/toolpad/connecting-to-data-sources/connections.md

```diff
@@ -0,0 +1,48 @@
+# Connections
+
+<p class="description">
+    <b>CONNECTION</b> instance allows us to reuse certain configuration details to create <b>queries</b> faster
+</p>
+
+In order to create a new `CONNECTION` instance press `+` button in the `Instance editor` on the left
```

Comment:
I wonder if we are not using the `<code>` element too frequently, that we have almost no cases where it should be used. I mean, considering https://www.notion.so/mui-org/Style-conventions-for-MUI-components-and-products-dff7d2d7d1ba4a4abd47f48cf345b800#8f86b8e3c0fb49e1b4df3276efb3848e and looking at https://stripe.com/docs/quotes/create, https://slack.com/help/articles/212906697-Create-a-user-group, https://docs.retool.com/docs/connect-api-resource, I would imagine:

```suggestion
In order to create a new connection instance press **+** button in the **Instance editor** on the left.
```

Thoughts? cc @samuelsycamore.

Reply:
My thinking behind using `code` was to create some sort of association that this is the main concept we are talking about and the most important part that one should memorise.

Very rough example - if I say "create a new connection" - is it immediately clear that we are talking about Toolpad connection instance or could it be some "mental connection" between elements? vs. if I say "create a new `connection`" I'm trying to teach that everything inside `code` is a "concept"/"action".

I'm fine with changing it all over the place, let's just first decide how. Whether it should remain just bolded or not singled out at all?

Reply:
Personally, I find the code blocks everywhere distracting, and it goes against my expectation of what I semantically associate with this markup. If emphasis is needed, I'd prefer it to be either **bold** or _italic_. If disambiguation is needed, a link could work.

Reply:
I would like to see these pages adhere to the guidelines established in our [Style guide](https://www.notion.so/mui-org/Style-guide-2a957a4168a54d47b14bae026d06a24b). There's no reason to give these words any special formatting.